### ISSUE TITLE
Could you update README to include simple lua remap syntax?

### ISSUE THREAD

**Issue by githorse** [junegunn/vim-easy-align#157]
I'm updating to the latest Neovim and building my config from scratch using lua. Gotta have my easy-align back! But offhand I'm not sure how you translate this:

```
" Start interactive EasyAlign in visual mode (e.g. vipga)
xmap ga <Plug>(EasyAlign)

" Start interactive EasyAlign for a motion/text object (e.g. gaip)
nmap ga <Plug>(EasyAlign)
```

into the new Lua syntax. I found an answer in [this issue](https://github.com/junegunn/vim-easy-align/issues/152), but unfortunately it's not working for me. In light of Neovim's escalating popularity, could the README be updated to show the Lua version of these mappings?

(Mentioning the `packer` install syntax would be nice too, but that's pretty straightforward so not a big deal.)

**Comment by junegunn** [comment]
I don't use Neovim, so I have no idea, but I wonder why you have to translate everything into Lua when Neovim is supposed to be compatible with Vim.

**Comment by githorse** [comment]
Yes, you're right, you can still use config files in Vimscript if you set it up correctly (I think), but I've decided to just commit to the new Neovim/Lua way instead of mixing the two. (Don't really know how to do that anyway.) 

Turns out this wasn't a very tricky translation. This seems to be working fine for me:

```
vim.keymap.set("n", "ga", "<Plug>(EasyAlign)")
vim.keymap.set("x", "ga", "<Plug>(EasyAlign)")
```
